keyboards/ocean/gin_v2/README.md

```diff
@@ -0,0 +1,24 @@
+# Gin V2
```

Comment:
Please rename the file so that it is lower case.

Reply:
done!